fireworks/flask_site/app.py

```diff
@@ -17,12 +17,11 @@
 from fireworks.flask_site.util import jsonify
 
 app = Flask(__name__)
-app.use_reloader = True
+# app.use_reloader = True
```

Comment:
What is the impact of disabled the reloader?

Reply:
As far as I remember the `use_reloader` is useful for debugging but by default, you would avoid it. This option is for continuously monitor the code of the module and restart the server when there is a change.  Usually, if the debug flag is set the server will automatically reload for code changes:
```python
app.run(debug=True, port=8080)
```

Reply:
Ok, I will defer to @computron on whether this is safe to comment out. But if it is then I would remove the entire line rather than commenting it.

Reply:
```suggestion

```